src/stdlib/downcase.rs

```diff
@@ -48,7 +44,7 @@ struct DowncaseFn {
 impl FunctionExpression for DowncaseFn {
     fn resolve(&self, ctx: &mut Context) -> Resolved {
         let value = self.value.resolve(ctx)?;
-        downcase(value)
```

Comment:
Just curious what clippy warning motivated this. I think it was easier to read before, but not a big deal.